_vector-search/vector-search-techniques/index.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Vector search techniques
+nav_order: 15
+has_children: true
+has_toc: false
+redirect_from:
+  - /search-plugins/knn/
+  - /search-plugins/knn/index/ 
+  - /vector-search/vector-search-techniques/     
+---
+
+# Vector search techniques
+
+OpenSearch implements vector search as *k-nearest neighbors*, or *k-NN*, search. k-NN search finds the k neighbors closest to a query point across an index of vectors. To determine the neighbors, you can specify the space (the distance function) you want to use to measure the distance between points.
+
+OpenSearch supports three different methods for obtaining the k-nearest neighbors from an index of vectors:
+
+- [Approximate search]({{site.url}}{{site.baseurl}}/search-plugins/knn/approximate-knn/)) (approximate k-NN, or ANN): Returns approximate nearest neighbors to the query vector. Usually, approximate search algorithms sacrifice indexing speed and search accuracy in exchange for performance benefits such as lower latency, smaller memory footprints, and more scalable search. For most use cases, approximate search is the best option.
+
+- Exact search: A brute-force, exact k-NN search of vector fields. OpenSearch supports the following types of exact search: 
```

Comment:
Should we add a brief section on exact search for symmetry with ANN?

Reply:
@jmazanec15 Could you please provide the information to create this section?